index.html

```diff
@@ -415,11 +416,10 @@
 
 <p>The <dfn>invoke idle callback timeout algorithm</dfn>:</p>
 <ol>
-<li>Let <var>callback</var> be the result of finding the entry in the <var>document</var>'s <a>list of idle request callbacks</a> or the <var>document</var>'s <a>list of runnable idle callbacks</a> that is associated with the value given by the <var>handle</var> argument passed to the algorithm.</li>
+<li>Let <var>callback</var> be the result of finding the entry in the <code>Window</code> object's <a>list of idle request callbacks</a> or the <a>list of runnable idle callbacks</a> that is associated with the value given by the <var>handle</var> argument passed to the algorithm.</li>
```

Comment:
I think this <code>Window</code > should be <var>window</var>, in order to be associated with the argument passed to the algorithm (same for the line below).